lib/rules/no-multi-spaces.js

```diff
@@ -70,7 +70,7 @@ module.exports = function(context) {
 
             var source = context.getSource(),
                 allComments = context.getAllComments(),
-                pattern = /[^\n\r\u2028\u2029 ] {2,}/g,  // note: repeating space
+                pattern = /[^\s].? {2,}/g,  // note: repeating space
```

Comment:
Not all whitespace is the same. There are four valid JS line breaks (seen in previous version of this file). It's best to keep those explicit and then do a space or tab.